packages/flutter/lib/src/material/app.dart

```diff
@@ -937,8 +962,8 @@ class _MaterialAppState extends State<MaterialApp> {
         cursorColor: effectiveCursorColor,
         child: AnimatedTheme(
```

Comment:
If `widget.themeAnimationStyle` is `noAnimation` could we just leave the AnimatedTheme out?